resources/testharness.js

```diff
@@ -520,6 +520,23 @@ policies and contribution forms [3].
             Object.prototype.toString.call(worker) == '[object ServiceWorker]';
     }
 
+    function get_test_name(func, name)
+    {
+        if (name) {
+            return name;
+        }
+
+        if (func) {
+            var func_code = func.toString();
+            var arrow = func_code.match(/^\(\)\s*=>\s*(.*)$/);
```

Comment:
Could we eat optional `{` in the regex? `func_code.match(/^\(\)\s*=>\s*({)?\s*(.?*)\s*$/);` and then eat the trailing `} if the first one matched, or something. And channeling @stephenmcgruer this could use some comments

Reply:
Done!